src/LanguageLine.php

```diff
@@ -30,17 +30,16 @@ public static function boot()
     public static function getTranslationsForGroup(string $locale, string $group): array
     {
         return Cache::rememberForever(static::getCacheKey($group, $locale), function () use ($group, $locale) {
-            return static::query()
+            $translations = [];
+
+            static::query()
                 ->where('group', $group)
                 ->get()
-                ->map(function (LanguageLine $languageLine) use ($locale) {
-                    return [
-                        'key' => $languageLine->key,
-                        'text' => $languageLine->getTranslation($locale),
-                    ];
-                })
-                ->pluck('text', 'key')
-                ->toArray();
+                ->map(function (LanguageLine $languageLine) use ($locale, &$translations) {
```

Comment:
Should have mentioned that I also really don't like references. Could you refactor this so `&` isn't needed?